docs/index.rst

```diff
@@ -30,51 +30,31 @@ Virtualenv
   :target: https://pypistats.org/packages/virtualenv
   :alt: Package popularity
 
-``virtualenv`` is a tool to create isolated Python environments. Since Python ``3.3``, a subset of it has been
-integrated into the standard library under the `venv module <https://docs.python.org/3/library/venv.html>`_. The
-``venv`` module does not offer all features of this library, to name just a few more prominent:
+``virtualenv`` is a tool to create isolated Python environments with its own copy of
+a working Python environment that is isolated from the rest of the system.
 
-- is slower (by not having the ``app-data`` seed method),
-- is not as extendable,
-- cannot create virtual environments for arbitrarily installed python versions (and automatically discover these),
-- is not upgrade-able via `pip <https://pip.pypa.io/en/stable/installing/>`_,
-- does not have as rich programmatic API (describe virtual environments without creating them).
+.. note::
+  Since Python ``3.3``, a subset of it has been integrated into the standard library under the `venv module <https://docs.Python.org/3/library/venv.html>`_. The
+  ``venv`` module does not offer all features of this library. See the :doc:`differences <differences_with_venv>` betwwen virtualenv and venv.
 
-The basic problem being addressed is one of dependencies and versions, and indirectly permissions.
-Imagine you have an application that needs version ``1`` of ``LibFoo``, but another application requires version
-``2``. How can you use both these libraries? If you install everything into your host python (e.g. ``python3.8``)
-it's easy to end up in a situation where two packages have conflicting requirements.
+The need for isolation of Python environments comes when there are two Python
```

Comment:
I prefer the initial formulation. I find the new sentences hard to follow and inaccurate in some places.

Reply:
This is adapted from https://docs.python.org/3/library/venv.html, and it seems readable to me.

Reply:
We'll have to agree to disagree on this then 😊